lib/fastifySession.js

```diff
@@ -78,25 +78,14 @@ function session (fastify, options, next) {
         })
         return
       }
-      if (options.rolling) {
```

Comment:
Btw, we didn't remove `rolling` from the docs.

This is how `express-session` uses rolling:
https://github.com/expressjs/session/blob/1010fadc2f071ddf2add94235d72224cf65159c6/index.js#L470-L472

Reply:
I know, we have to reimplement or check if rolling is working as expected.

Reply:
Cool I made an issue: https://github.com/fastify/session/issues/148